content/09.evolution.md

```diff
@@ -96,11 +96,47 @@ IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their
+Although the genome-wide sequence identity was lower between SARS-CoV-2 and this pangolin virus than BatCoV-RaTG13, its particularly high similarity in the receptor binding domain (RBD) of the spike (_S_) gene with SARS-CoV-2 drew further attention [@doi:10.1038/s41586-020-2169-0; @doi:10.1016/j.cub.2020.03.022].
+The SARS-CoV-2 RBD differs from the pangolin coronavirus RBD by only one amino acid change [@doi:10.1016/j.cub.2020.03.022], and the sequence identity between the regions is 97.4% [@doi:10.1038/s41586-020-2169-0].
+Pangolins were therefore identified as a potential intermediate host of SARS-CoV-2 between bats and humans.
+
+However, data collected from May 2017 to November 2019 by a research team interested in tick-borne illnesses identified no bats or pangolins sold at these markets leading up to the emergence of COVID-19 [@doi:10.1038/s41598-021-91470-2].
+Additionally, endemic bat species are typically in hibernation at the time that SARS-CoV-2 emerged [@doi:10/ggjr43].
+Therefore, it is possible that animals associated with these markets were infected by bats, but it is not clear whether the disease emerged in a different location and/or whether it is associated with a different species.
+There were 38 species observed at the market in the 2.5 years leading up to the emergence of SARS-CoV-2, indiciating significant diversity in the animals with which humans were interacting [@doi:10.1038/s41598-021-91470-2].
+As with SARS-CoV-1, the species of origin for SARS-CoV-2 therefore remains unresolved.
+
+Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
+However, potentially due to public misunderstanding about recombination and complex evolutionary processes like coevolution, the similarity to pangolin _S_ has resulted in popular conspiracy theories that the virus did not arise naturally.
+The similarity of _S_ to that of pangolin viruses could arise from either recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153], rather than requiring human intervention.
+Such suspicions may also have been fueled, in part, by the lack of well-characterized bat coronaviruses which means that SARS-CoV-2 is still relatively derived from documented coronaviruses surveyed in bats [@doi:10.1016/j.tree.2020.12.002].
+While it has been suggested that more thorough investigation of the origins of COVID-19 may have some value [@doi:10.1126/science.abj0016], in many cases, support for the "lab-leak" theory is politically motivated [@doi:10.1038/d41586-021-01383-3].
+A more robust panel of zoonotic viruses against which to compare SARS-CoV-2 would allow for conclusive dismissal of these politicized claims, underscoring another potential benefit of more thorough monitoring of zoonotic diseases.
```

Comment:
```suggestion
In general, approaching the origination of SARS-CoV-2 to develop proactive approaches to manage future emerging pathogens would be expected to provide more value than a reactive interest in a pathogen that has already caused a pandemic.
A more robust panel of zoonotic viruses against which to compare SARS-CoV-2 would allow for conclusive dismissal of these politicized claims, underscoring another potential benefit of more thorough monitoring of zoonotic diseases.
```

Reply:
@RLordan thinking about this, I feel like maybe it's worth pointing out that proactive interest in the origins of SARS-CoV-2 is good, reactive blame-games are not where we need to be investing our energy or this will happen again. As I keep telling my father-in-law, I would be delighted if people genuinely cared about and valued biosafety standards, but that isn't the vibe I get from the articles he sends me :laughing:

Reply:
I totally agree with you, I have my dad in my family to send me those perspectives haha 

I would agree on those perspectives, there is too much fingerprinting, and not enough logic. How do you write that politely haha